test_scripts/Polices/External_UCS/External_Consent_ON/013_ATF_Policies_External_Consent_ON_through_ignition_cycles_disallowed_by_external_consent_entities_omitted.lua

```diff
@@ -0,0 +1,163 @@
+---------------------------------------Preconditions--------------------------------------------------
+------------------------------------------------------------------------------------------------------
+-- Start SDL and register application
+common_functions_external_consent:PreconditonSteps("mobileConnection","mobileSession")
+-- Activate application
+common_steps:ActivateApplication("Activate_Application_1", config.application1.registerAppInterfaceParams.appName)
+------------------------------------------------------------------------------------------------------
+------------------------------------------- Functions ------------------------------------------------
+------------------------------------------------------------------------------------------------------
+local function CheckGroup001IsConsented()
+  --------------------------------------------------------------------------
+  -- Main check:
+  -- RPC of Group001 is disallowed to process.
+  --------------------------------------------------------------------------
+  Test[TEST_NAME_ON .. "MainCheck_RPC_of_Group001_is_disallowed"] = function(self)
+    local corid = self.mobileSession:SendRPC("SubscribeVehicleData",{rpm = true})
+    self.mobileSession:ExpectResponse(corid, {success = false, resultCode = "DISALLOWED"})
+    EXPECT_NOTIFICATION("OnHashChange")
+    :Times(0)
+  end
+end
+
+local function IgnitionOffOnActivateApp(test_case_name)
+  common_steps:IgnitionOff("Precondition_Ignition_Off_" .. test_case_name)
+  common_steps:IgnitionOn("Precondition_Ignition_On_" .. test_case_name)
+  common_steps:AddMobileSession("Precondition_Add_Mobile_Session_1_" .. test_case_name, "mobileConnection","mobileSession")
+  common_steps:RegisterApplication("Precondition_Register_Application_1_" .. test_case_name, "mobileSession", config.application1.registerAppInterfaceParams)
+  common_steps:ActivateApplication("Precondition_Activate_Application_1_" .. test_case_name, config.application1.registerAppInterfaceParams.appName)
+  Test["Precondition_Wait_For_Database_After_Activate_Application_1_" .. test_case_name] = function(self)
+    common_functions:DelayedExp(5000)
+  end
+end
+
+------------------------------------------------------------------------------------------------------
+------------------------------------------ Tests -----------------------------------------------------
+------------------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------
+-- TEST 10:
+-- In case
+-- SDL has received SDL.OnAppPermissionConsent ("externalConsentStatus: ON") from HMI
+-- SDL must
+-- use this value through ignition cycles
+-- until this externalConsentStatus value is changed by corresponding notification from HMI.
+--------------------------------------------------------------------------
+-- Test 10.03:
+-- Description:
+-- 1. disallowed_by_external_consent_entities_off.
```

Comment:
@dboltovskyi, regarding description in test set, disallowed_by_external_consent_entities_on should be omitted. Please update.